docs/user/unsupported/process/process-feature.rst

```diff
@@ -59,4 +59,22 @@ This is a very flexible process which can be used to:
      description: String   description                    description: String
                            area=area( the_geom)           area: Double
                            text=concatenate(name,'-',id)  text: String
- 
+
+
+
+Filtering Features
```

Comment:
Tad too generic, it's not just filtering, it's more like "choosing a particular feature in a group". "Group candidate selection" maybe?

Reply:
fixed

Reply:
The title still looks the same to me?

![image](https://user-images.githubusercontent.com/325433/97695844-f6a68900-1aa4-11eb-8ea7-0d82d6210ce0.png)

Reply:
fixed